public/main.html

```diff
@@ -119,7 +111,7 @@ <h2 class="start-main-content-title">Добрый вечер</h2>
           </a>
           <a class="main-playlist" href="">
```

Comment:
href не может быть пустым
Должно быть хоть какое-то значение.
Например
```suggestion
          <a class="main-playlist" href="#">
```

Reply:
Я понимаю, видимо во время разработки забыл в некоторых местах указать